tsconfig.json

```diff
@@ -1,17 +1,37 @@
 {
   "compilerOptions": {
-    "target": "es2019",
-    "module": "esnext",
-    "moduleResolution": "Node",
-    "strict": true,
-    "jsx": "preserve",
-    "importHelpers": true,
-    "allowJs": true,
-    "skipLibCheck": true,
     "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
+    "target": "es2021",
+    "verbatimModuleSyntax": true,
+
+    "allowJs": true,
     "resolveJsonModule": true,
-    "sourceMap": true,
+    "moduleDetection": "force",
+
+    "strict": true,
+    "noUncheckedIndexedAccess": true,
+
+    "moduleResolution": "Node",
+    "module": "ESNext",
+    "noEmit": true,
```

Comment:
How about adding `sourcemap: true,`? I did not test it myself but would like to be able to debug the original files on the productive/test systems.